components/kyma-environment-broker/common/orchestration/strategies/parallel.go

```diff
@@ -83,6 +83,10 @@ func (p *ParallelOrchestrationStrategy) Insert(execID string, operations []orche
 			p.handleRescheduleErrorOperation(execID, &operations[i])
 			p.log.Errorf("while processing operation %s: %v, will reschedule it", op.ID, err)
 		} else {
+			if p.dq[execID].ShuttingDown() {
+				return fmt.Errorf("the execution ID %s is shutdown", execID)
+			}
```

Comment:
Hi @ebensom  this is appeared when the orchestration is the latter phase of `in progress` and not  achieves the final  status (e.g.`succeed` / `failed` ), at this moment if user quickly execute `kcp o *** retry`, it may enter the this scenario. I will move it out of the loop in next PR.